apps/addons/models.py

```diff
@@ -508,12 +512,15 @@ def initialize_addon_from_upload(cls, data, is_listed=True):
         if not locale_is_set:
             addon.default_locale = to_language(trans_real.get_language())
 
+        addon.save()
+        if old_guid_addon:
+            old_guid_addon.update(guid='guid-reused-by-pk%s' % addon.pk)
```

Comment:
I thought .format was recommended for messages and % was okay for other token substitution. But sure, can do.

There is no issue with long guids until we get to 10^236 addons on AMO ;-)